apps/cuda_mat_mul/mat_mul_generator.cpp

```diff
@@ -28,30 +29,39 @@ class MatMul : public Halide::Generator<MatMul> {
 
         Func prod("prod");
         RDom r(0, size);
```

Comment:
We should do something interesting here, like use `RDom r(A.dim(0).min(), A.dim(0).extent())`

In the schedule, do something interesting like: `RVar ro; prod.update().split(r, ro, r, 64).unroll(r, 16);`

Hopefully that would give an unrolled loop of 4 wmma instructions, and also include a non-wmma tail loop to handle the remainder of the reduction.

Reply:
Edited that because I realized the proposed schedule was wrong

Reply:
I wasn't sure how to proceed with this but I will try this schedule